script/1_0_2current_en_script/Arcueid/Day 3/03_00_ARC03_5B.txt

```diff
@@ -81,12 +81,12 @@ All I want right now is to be left alone in my room, take off my glasses, close
 "Haa――――haa――――haa――――"
 @gThe woman is walking slowly.
 @gShe's not interested in me at all.
-@g―――A faint question crossed my mind.
-@gNot interested? That's absurd. Is that really the case? But the fact is, she's not aware of me at all. Did something happen? There must have been something. She's probably very tired. Exhausted. Drained. That's why there is an opportunity. The reason I feel this way is because, when all is said and done, I'm confident that /I can ■ her/.
+@g―――A faint question crosses my mind.//@g　―――かすかな疑問が脳裏をかすめる。
+@gNot interested? That's absurd. Is that really the case? But the fact is, she's not aware of me at all. Did something happen? There must have been something. She's probably very tired. Exhausted. Drained. That's why there's this opportunity. The reason I feel this way is because, when all is said and done, I'm confident that /I can ■ her/.
```

Comment:
Candidate for blank line removal in allscr (ignore messed up fonts)
![2021-12-22-010530_1013x625_scrot](https://user-images.githubusercontent.com/2476799/147017466-1260cb88-70a1-420a-bd1c-d497b9980f18.png)